app/models/scheme.rb

```diff
@@ -5,6 +5,9 @@ class Scheme < OpenStruct
   extend GdsApi::Helpers
 
   FACET_KEYS = [:areas, :business_sizes, :locations, :sectors, :stages, :support_types]
+  # This list should stay in sync with Publisher's AREA_TYPES list
+  # (https://github.com/alphagov/publisher/blob/master/app/models/area.rb#L7).
```

Comment:
Can you add a comment in Publisher to make it clear that the list needs to be kept in sync between Applications?